lib/src/utils/mod.rs

```diff
@@ -83,3 +81,69 @@ pub async fn log_slots(client: &AzeClient, account_id: AccountId) {
         println!("Account slot {:?} --> {:?}", i, regular_account.storage().get_item(i));
     }
 }
+
+pub async fn initial_setup(mut client: &mut AzeClient) -> (FungibleAsset, AccountId, AccountId, GameStorageSlotData){
+    let small_blind_amt = 5u8;

```

Comment:
Let's make these values as default constants and store it in `lib/src/constant` modules